controllers/openstack_ansibleee_controller.go

```diff
@@ -495,7 +497,12 @@ func addCmdLine(
 	var cmdLineEnvVar corev1.EnvVar
 	var err error
 	cmdLineEnvVar.Name = "RUNNER_CMDLINE"
-	cmdLineEnvVar.Value = "\n" + instance.Spec.CmdLine + "\n\n"
+	// If we are in a debug mode we set command to `sleep 0`
+	if instance.Spec.Debug {
+		cmdLineEnvVar.Value = "\nsleep 0\n\n"
```

Comment:
won't sleep 0 just exit? did you intend sleep infinity?

Reply:
It should get the operator stuck in Active state, preventing outright success.

Reply:
I've set the sleep time span to 1 day.